docker-compose-dev.yml

```diff
@@ -23,5 +23,24 @@ services:
     ports:
       - 6379:6379
 
+  broker:
```

Comment:
question: should this not be part of the main `docker-compose.yml` in that it is useful whether you run everything in Docker, or partially

Reply:
Good shout, added to the main `docker-compose.yml` file also.